tap_github/repository_streams.py

```diff
@@ -519,6 +519,159 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
     ).to_dict()
 
 
+class MilestonesStream(GitHubRestStream):
+    name = "milestones"
+    path = "/repos/{org}/{repo}/milestones"
+    primary_keys = ["id"]
+    replication_key = "updated_at"
+    parent_stream_type = RepositoryStream
+    state_partitioning_keys = ["repo", "org"]
+    ignore_parent_replication_key = True
+
+    schema = th.PropertiesList(
+        # Parent Keys
+        th.Property("repo", th.StringType),
+        th.Property("org", th.StringType),
+        # Rest
+        th.Property("url", th.StringType),
+        th.Property("html_url", th.StringType),
+        th.Property("labels_url", th.StringType),
+        th.Property("id", th.IntegerType),
+        th.Property("node_id", th.StringType),
+        th.Property("number", th.IntegerType),
+        th.Property("state", th.StringType),
+        th.Property("title", th.StringType),
+        th.Property("description", th.StringType),
+        th.Property(
+            "creator",
+            th.ObjectType(
+                th.Property("login", th.StringType),
+                th.Property("id", th.IntegerType),
+                th.Property("node_id", th.StringType),
+                th.Property("avatar_url", th.StringType),
+                th.Property("gravatar_id", th.StringType),
+                th.Property("url", th.StringType),
+                th.Property("html_url", th.StringType),
+                th.Property("followers_url", th.StringType),
+                th.Property("following_url", th.StringType),
+                th.Property("gists_url", th.StringType),
+                th.Property("starred_url", th.StringType),
+                th.Property("subscriptions_url", th.StringType),
+                th.Property("organizations_url", th.StringType),
+                th.Property("repos_url", th.StringType),
+                th.Property("events_url", th.StringType),
+                th.Property("received_events_url", th.StringType),
```

Comment:
I'd be of the opinion that all these fields can easily be guessed from username alone, and so may not be worth storing (they are long strings). For all other streams, we've kept these to the non-guessable part, factored out in #98 for easy reuse.